tests/integration_tests/security/test_jail.py

```diff
@@ -282,7 +282,7 @@ def test_cgroups(test_microvm_with_initrd, sys_setup_cgroups):
     test_microvm.jailer.numa_node = 0
     test_microvm.jailer.cgroup_ver = sys_setup_cgroups
     if test_microvm.jailer.cgroup_ver == 2:
-        test_microvm.jailer.cgroups = ['cpu.weight=2']
```

Comment:
This change looks unrelated.

Reply:
It's not unrelated, it is meant to test the fix and ensure that controllers' properties that themselves contain dots (`weight.nice`) do not raise an error, like it did before the fix.